template/lambda-sqs-worker-cdk/src/framework/logging.ts

```diff
@@ -4,11 +4,16 @@ import createLogger, { createDestination } from '@seek/logger';
 
 import { config } from 'src/config.js';
 
-interface LoggerContext {
+interface LambdaContext {
   awsRequestId: string;
 }
 
-export const loggerContext = new AsyncLocalStorage<LoggerContext>();
+interface RecordContext {
+  sqsMessageId: string;
+}
+
+export const lambdaContext = new AsyncLocalStorage<LambdaContext>();
+export const recordContext = new AsyncLocalStorage<RecordContext>();
```

Comment:
I decided to add a second context here as I think it's important to correlate logs related to a single record since we are now doing partial batch failure